tests/augmentation/test_augmentation.py

```diff
@@ -4071,6 +4071,20 @@ def test_same_on_batch(self, device, dtype):
         output_tensor = transform(input_tensor)
         self.assert_close(output_tensor[0], output_tensor[1])
 
+    @pytest.mark.slow
+    @pytest.mark.skipif(
+        torch_version_le(1, 13, 1),
+        reason="torch.compile is not available in previous versions",
+    )
+    def test_compile(self, device, dtype):
```

Comment:
Just to keep the same test name as in other apis, also uses the `torch_optimizer` to skip the necessary cases
```suggestion
    def test_dynamo(self, device, dtype, torch_optimizer):
```

Reply:
torch_optimizer is used for something?

Reply:
The `torch_optimizer` is the `torch.compile` callable itself (check here: https://github.com/kornia/kornia/blob/22c40fbfbf74160e98c6d1d6c42f90d915d6e7f2/conftest.py#L77) -- inside of this fixture are the checks around the torch compile are available or not -- if you're willing, the best thing would be to move this checker heuristic to a function within `testing/` so we can just use it in the skip marker instead of using the fixture

Reply:
for cases like these methods compile for the augmentations API, I will want to move to use the `optimizer_backend` fixture which we can inject like the `device` and `dtype` 

right now we are just testing the inductor backend compiler, but in the future we may want to move to another backend

Reply:
and here is a test which uses the `torch_optimizer` fixture -- https://github.com/kornia/kornia/blob/22c40fbfbf74160e98c6d1d6c42f90d915d6e7f2/tests/filters/test_blur.py#L134

Reply:
Understood! Thanks for the explanation!